web/static/css/styles.css

```diff
@@ -0,0 +1,210 @@
+    > div.container > div.flex-row {
+      > div.social-media-links
+        > div.official-collapsible-content
+        > button.btn-official {
+        border: none;
+        font-size: 0.875rem;
+        background: transparent;
+        text-align: left;
+        text-wrap: nowrap;
+        &[aria-expanded="true"] {
+          > span.ca-gov-icon-caret-up {
+            display: inline-block;
+          }
+          > span.ca-gov-icon-caret-down {
+            display: none;
+          }
+        }
+        &[aria-expanded="false"] {
+          > span.ca-gov-icon-caret-up {
+            display: none;
+          }
+          > span.ca-gov-icon-caret-down {
+            display: inline-block;
+          }
+        }
+      }
+      @media (width <= 770px) {
+        flex-direction: column !important;
+        > div.settings-links {
+          margin-left: 0 !important;
+          justify-content: flex-start;
+          > a {
+            margin-left: 0;
+            margin-right: 1rem;
+            font-size: 0.8rem;
+          }
+        }
+      }
+      @media (width <= 350px) {
+        > div.social-media-links {
+          flex-direction: column;
+        }
+      }
+    }
+  }
+  > div.section-default > div.branding > div.header-organization-banner {
+    display: flex;
+    margin-right: 0;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    > a > div.logo-assets {
+      > img.logo-img {
+        max-height: 60px;
+        height: 60px;
+        aspect-ratio: 18/5;
+      }
+      > div.logo-text {
+        > span.color-primary {
+          color: var(--color-p2, #046b99);
+        }
+        > span.logo-state {
+          font-size: 1.4rem;
+          font-weight: 700;
+          position: relative;
+          top: -5px;
+        }
+        > span.logo-dept {
+          font-size: 2.4rem;
+          height: 2.5rem;
+        }
+      }
+      @media (width <= 991px) {
+        > img.logo-img {
+          max-height: 50px;
+        }
+        > div.logo-text {
+          > span.logo-state {
+            font-size: 1.3rem;
+            top: -2px;
+          }
+          > span.logo-dept {
+            font-size: 1.8rem;
+            height: 2.2rem;
+          }
+        }
+      }
+    }
+    @media (width <= 575px) {
+      margin-right: 15%;
+      > a > div.logo-assets {
+        > div.logo-text {
+          > span.logo-dept {
+            font-size: 1.6rem;
+            height: 2.2rem;
+          }
+          > span.logo-state {
+            font-size: 1rem;
+            position: relative;
+            top: -2px;
+          }
+        }
+        > img.logo-img {
+          max-height: 50px;
+        }
+      }
+    }
+  }
+  > div.mobile-controls > div.main-nav-icons {
+    top: -5rem;
+    > button.toggle-menu {
+      margin-top: 1rem;
+    }
+  }
+  > div.navigation-search {
+    > div.nav-drawer {
+      padding-bottom: 0;
+    }
+    > div.search-container > div.d-block > a.drawer-logo-link {
```

Comment:
Appears to be unused

Reply:
@thekaveman needed. I will update HTML later

Reply:
@kkoryaka I understand we may need it later, and I'm definitely not opposed to adding this CSS back when we do need it. 

It's so easy for this kind of thing to get forgotten about though, and then we just have dead code floating around for no reason. I'd rather start clean if we can.

Reply:
@thekaveman Added the nav drawer logo HTML. So, these styles should be used now